### PR TITLE
Hook up dag and tools table - first pass

### PR DESCRIPTION
**Pull request by denis-yuen**
* Had to change interface a little
* Mutated the output somewhat, looks like we support/use a specific Cytoscape output format that looks like http://manual.cytoscape.org/en/stable/Supported_Network_File_Formats.html#cytoscape-js-json
* some small testing

### CODE REVIEW

File: src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java [hunk 1]
```diff
@@ -61,11 +66,26 @@ public String launchInstructions(String trsID) {
...
-        String contents,
-        Map<String, Pair<String, GenericFileType>> indexedFiles) {
-      throw new UnsupportedOperationException();
+    public List<RowData> generateToolsTable(String initialPath, String contents, Map<String, Pair<String, GenericFileType>> indexedFiles) {
+      final Map<String, Object> workflow = loadWorkflow(contents);
+      final Map<String, Object> elements = Cytoscape.getElements(workflow);
+      final List<Map> nodes = (List<Map>)elements.getOrDefault("nodes", Lists.newArrayList());
+      return nodes.stream().map(node -> {
+        final RowData rowData = new RowData();
+        final Map<String, Object> nodeData = (Map<String, Object>)node.get("data");
```
Comment (coverbeck) [inline src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java]: Probably so, but just checking -- is `nodeData` guaranteed to not be `null`?

Comment (denis-yuen) [reply src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java]: Looks like yes
https://github.com/galaxyproject/dockstore-galaxy-interface/pull/10/files#diff-67da40517c8bf816074537fcf24ef99cR81

**Comment by jmchilton** [comment]
Awesome - thanks so much! Anything else I can do to help at this time? Maybe work on generating those Docker images (that will be pretty hard though).

**Comment by denis-yuen** [comment]
> Awesome - thanks so much! Anything else I can do to help at this time? Maybe work on generating those Docker images (that will be pretty hard though).

Not yet, but soon we'll probably have a few entries on our dev server for you to give feedback, start seeing where we're using the wrong terminology, etc.

**Pull request merged**